server/test/controllers/articles.spec.js

```diff
@@ -86,7 +86,7 @@ describe('API endpoint /articles/', () => {
       expect(response.body.data.message).to.eqls('Field validation error');
     });
 
-    it('It should be able to handle unexpected errors thrown when creating articles', async () => {});
+    it('It should be able to handle unexpected errors thrown when creating articles', async () => { });
```

Comment:
This test case line should not be here since for some reason there are no tests written to cover what it describes.